reflection_refraction.py

```diff
@@ -0,0 +1,136 @@
+from distutils.util import strtobool
+import math
+from matplotlib import pyplot as plt
+
+distance = 5
+
+# Introduction
+def intro():
+    print("\nReflection and refraction of light Calculator")
+    print("By Martin Lambov")
+    print("GitHub: https://github.com/5neepy")
+
+# Check if the equetion is posible
+def check_if_posible(alfa, n1, n2):
```

Comment:
Instead of comments clarifying the purpose of the function, on usually writes doc-strings:
```suggestion

def check_if_posible(alfa, n1, n2):
    '''Check if the reflection/refraction conditions are met.
    
    :param alfa: ...
    :param n1: ...
    :param n2: ...
    '''
```

Reply:
I don't get it

Reply:
The idea of your comment `# Check if the equetion is posible` is to make clear what the method is doing, right? 

The same is the idea of the doc-string:
```
    '''Check if the reflection/refraction conditions are met.
    
    :param alfa: ...
    :param n1: ...
    :param n2: ...
    '''
```

However, the format of the doc-string is fixed and one can use the doc-strings to auto-generate a documentation for the source code. 

Check [this](https://stackoverflow.com/a/24385103) out.

Reply:
My suggestion here would be to right for all your functions the corresponding doc-strings.

Reply:
Oh ok. I will do it.